composer.json

```diff
@@ -37,5 +37,15 @@
         "sort-packages": true
     },
     "extra": {
+      "enable-patching": true,
```

Comment:
As far as I know, "enable-patching" is only relevant for [package level](https://github.com/cweagans/composer-patches#allowing-patches-to-be-applied-from-dependencies) (AKA root) composer.json files.  [localgovdrupal/localgov_project](https://github.com/localgovdrupal/localgov_project/blob/65742666e3699bbda6fd72ede9911f32ffb72a22/composer.json#L38) already has ```enable-patching``` set to ```true``` making this a bit redundant :(